src/utils/tests/formValidator.test.js

```diff
@@ -1,184 +0,0 @@
-import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
```

Comment:
Could you clarify what is the reason to delete that file?

Reply:
It's not deleted, but just renamed :)
This is a way how Github shows it